web.d

```diff
@@ -4005,7 +4008,7 @@ class NotFoundException : Exception
 struct ListenConfig
 {
 	string addr;
-	ushort port = 80;
+	ushort port = 8000;
```

Comment:
So is it 8000 or 8080?

If anyone other than me actually used DFeed, I would say "don't change the defaults, that might actually break someone's setup", but it doesn't really matter.